### PR TITLE
fix(Apple Music): broken on beta website

### PR DESCRIPTION
**Pull request by down-bad**
## Description 

Fix songs not showing

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by ImRodry** [comment]
Can we get a before and after?

**Comment by down-bad** [comment]
image
if you're on premid 2.2.3 or lower, otherwise nothing

File: websites/A/Apple Music/presence.ts [hunk 1]
```diff
@@ -27,9 +27,19 @@ presence.on("UpdateData", async () => {
 			?.querySelector<HTMLVideoElement>("video#apple-music-video-player");
 	if (video?.title || audio?.title) {
 		const media = video || audio,
-			timestamp = document.querySelector<HTMLInputElement>(
-				"input[aria-valuenow][aria-valuemax]"
-			),
+			timestamp =
+				document.querySelector<HTMLInputElement>(
+					"input[aria-valuenow][aria-valuemax]"
+				) ||
```
Comment (ImRodry) [inline websites/A/Apple Music/presence.ts:33]:
```suggestion
				) ??
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

**Review by ImRodry — APPROVED** [review]

**Comment by down-bad** [comment]
Apparently the progress bar doesn't update if the website is in the background, so when the song changes it keeps the timestamp from the previous one. That's awesome...

**Comment by ImRodry** [comment]
Not much you can do about that then

**Comment by down-bad** [comment]
We can get the timestamp from the audio element

**Review by ImRodry — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**